src/Auth_Command.php

```diff
@@ -285,292 +562,22 @@ public function delete( $args, $assoc_args ) {
 	 *
 	 */
 	public function list( $args, $assoc_args ) {
```

Comment:
Option to list and view whitelisted ips is missing. Handle it in `list` function with `--ip/whitelist` flag or something similar.